src/components/MisakaWrite/LoginButton.vue

```diff
@@ -0,0 +1,28 @@
+<script lang="ts" setup>
+import { Icon } from '@iconify/vue';
+</script>
+
+<template>
+  <MisakaLogin :on="on" @LoginSwich="But"></MisakaLogin>
```

Comment:
For better readability and maintainability, change
- `MisakaLogin` to `Login`
- `on` to `visible`
- `But` to `toggle_visibility`